runtime/queries/javascript/textobjects.scm

```diff
@@ -0,0 +1,36 @@
+(function_declaration
+  body: (statement_block)) @function.around
+
+(function
+  body: (statement_block)) @function.around
+
+(export_statement
+  declaration: (function_declaration) @function.around) 
+
+(arrow_function
+  body: (statement_block)) @function.around
```

Comment:
Instead of `(statement_block)` let's match on `(_)` in these rules. This allows capturing single-expression arrow functions like `x => x + 1`. That node can also me barked as `function.inside` which is usually used for function bodies:

```scm
(function body: (_) @function.inside) @function.around
```